decidim-proposals/app/controllers/decidim/proposals/admin/proposals_controller.rb

```diff
@@ -32,6 +32,45 @@ def create
           end
         end
 
+        def update_category
+          authorize! :update, Proposal
+          if params[:proposal_ids].present? && params[:category][:id].present?
+            category = Decidim::Category.find_by id: params[:category][:id]
+            updated = { oks: [], invalids: [] }
+
+            Proposal.where(id: params[:proposal_ids]).find_each do |proposal|
```

Comment:
I'd rather move all this logic to the command

Reply:
OK, now the `UpdateProposalCategory` command will expect `category_id` and `proposal_ids`